src/ripple/protocol/impl/TER.cpp

```diff
@@ -101,6 +101,7 @@ transResults()
         MAKE_ERROR(tefINVARIANT_FAILED,       "Fee claim violated invariants for the transaction."),
         MAKE_ERROR(tefTOO_BIG,                "Transaction affects too many items."),
         MAKE_ERROR(tefNO_TICKET,              "Ticket is not in ledger."),
+        MAKE_ERROR(tefSEQ_AND_TICKET,         "Transaction contains a TicketSequence and a non-zero Sequence."),
```

Comment:
Should this be a `tem`? `tef` usually means that the problem is in an interaction with the ledger, whereas this happens because a tx should never have both `TicketSequence` and `Sequence`